collector/src/main/java/com/bitmovin/analytics/utils/Util.java

```diff
@@ -48,7 +48,7 @@ public static String getUserId(Context context) {
                 Settings.Secure.ANDROID_ID);
     }
 
-    public static long getElapsedTimestamp() {
+    public static long getElapsedTime() {
```

Comment:
would make sense to add a description here so every one knows what this method and time unit is about
```
return the time since the system was booted, and guaranteed to be monotonic
Details here: https://developer.android.com/reference/android/os/SystemClock
```